docs/mosviz/import_data.rst

```diff
@@ -1,6 +1,31 @@
+.. _mosviz-import-data:
+
 ***********
 Import Data
 ***********
 
-Words go here...
+Currently, data must be imported into Mosviz using the API in the Jupyter notebook. 
+After initializing the app as explained in :ref:`mosviz-notebook`, 
+you must generate three lists containing the filenames for the 1D spectra, 
+2D spectra, and images in your dataset. These three lists are taken as arguments 
+to the Mosviz `load_data` method. An example is given below, where `file_dir` is a 
+directory that contains all the files for the dataset to be loaded. 
```

Comment:
This line needs to end with a colon in order for the following lines to count as code.

Reply:
Just rebased and pushed an update with this change.